app/UiTPAS/Slack/UdbOrganizerMessageBuilder.php

```diff
@@ -0,0 +1,35 @@
+<?php
+
+declare(strict_types=1);
+
+namespace App\UiTPAS\Slack;
+
+use App\Domain\Integrations\Environment;
+use App\Domain\Integrations\Integration;
+use App\Domain\Integrations\UdbOrganizer;
+
+final readonly class UdbOrganizerMessageBuilder
```

Comment:
There seems to be a lot of common code with the existing `SlackMessageBuilder`

Reply:
Not really? I did look into reusing the existing `SlackMessageBuilder` but the main reason i decided not to use it is because the inferface requires an `Integration`, but in my case I start from an `UdbOrganizer`.
So just making another implementation was difficult.

I can however, merge both classes into 1 if you prefer, with a second method?
Don't really see the benefit, but that is an option.

Reply:
Now I also see this class does not derive from `MessageBuilder`.

But I still think there is a lot in common, for example:
- Message title
- Type
- Status
- Subscription
- Website
- Open in publiq-platform

The only extra things are open in UiTPAS and open in UDB3

Reply:
There is also nothing wrong with adding an extra method `toMessageWithOrganizer` and implementing that inside the existing `SlackMessageBuilder`. That is also one of the reasons this concrete `SlackMessageBuilder` is inside the `Notifications` namespace.